.gitattributes

```diff
@@ -1 +1,4 @@
 * text=auto eol=lf
+/*/**/Dockerfile            linguist-generated
+/*/**/docker-entrypoint.sh  linguist-generated
```

Comment:
Do we need this here?

Reply:
We don't need it no, it's something we do in other Docker Official Images to help in reviews. I can take it out if the rest of the team would prefer not having it

Reply:
My preference would be to split it into its own PR(topic) as it has nothing to do with "Add Node debian core variant" ;)

Reply:
done